cadc-vos-server/src/main/java/org/opencadc/vospace/server/pkg/VospacePackageRunner.java

```diff
@@ -122,6 +125,7 @@ public class VospacePackageRunner extends PackageRunner {
     private PathResolver pathResolver;
     private List<URI> targetList;
     private String appName;
+    private String baseNodeUri;
```

Comment:
String type and "Uri" in the name is a bad sign here

Reply:
I usually use xxxURI if the type is URI, and xxxUri for a string that is a URI, but of course no one else would know that. Updated the name.

Reply:
And as it turns out, not used. I think that was the first pass at figuring out if a target was in the package tree. Removed.